runtime/runtime-domain-service/ballerina/APIClient.bal

```diff
@@ -1659,6 +1730,9 @@ public class APIClient {
                     string httpHeader = <string>policyParameters.get("headerName");
                     removePolicies.push(httpHeader);
                 }
+                if (policyName == "interceptor") {
```

Comment:
is this block for future implementation?

Reply:
Removed using https://github.com/wso2/apk/pull/1019/commits/8bf9dbd72a7f7e41c97d3d622fae536a8b3791ed